os/include/tinyara/net/net.h

```diff
@@ -174,6 +174,7 @@ struct socket {
 	/** sockets currently are built on netconns, each socket has one netconn */
 	void *conn;
 	sock_type type; // lwip_sock, uds, ...
+	void *sock;
```

Comment:
Please add a comment to explain the meaning of this variable.

Reply:
done